extensions/applicationinsights-analytics-js/Tests/Unit/src/AnalyticsPlugin.tests.ts

```diff
@@ -1063,324 +1063,329 @@ export class AnalyticsPluginTests extends AITestClass {
         });
 
 
-        this.testCaseAsync({
+        this.testCase({
             name: "OnErrorTests: _onerror logs name of unexpected error thrown by trackException for diagnostics",
-            stepDelay: 1,
             useFakeTimers: true,
-            steps: [() => {
-                // setup
-                const sender: Sender = new Sender();
-                const core = new AppInsightsCore();
+            test: () => {
+                return this._asyncQueue().add(() => {
+                    // setup
+                    const sender: Sender = new Sender();
```

Comment:
The first step of each test should be part of the initial test function, and at the end of this step it should then use
```ts
return this._asyncQueue().add(() => <subsequent step>)
```
So that all subsequent steps have access to the variables and values from this initial step for any additional context